### PR TITLE
Bump xDSL to 0.11

### PR DESCRIPTION
**Pull request by PapyChacal**
This PR updates devito (in the stencil_mad_dash branch) to work with xDSL 0.11.

### CODE REVIEW

**Review by georgebisbas — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -18,5 +18,5 @@ distributed<2022.10
 pytest>=3.6,<8.0
 pytest-runner
 pytest-cov
-xdsl<=0.9.2
+xdsl<=0.12
```
Comment (AntonLydike) [inline requirements.txt:21]: I would suggest `~= 0.12`. Using `<=0.12` will not trigger pip to upgrade older versions.

**Review by AntonLydike — APPROVED** [review]
Looks good, except for the requirements file version constraint

File: requirements.txt (hunk 1 above)
Comment (AntonLydike) [reply requirements.txt:21]: Or even just `==0.12` because everything breaks with every update anyway

Comment (PapyChacal) [reply requirements.txt:21]: Using ~= for flexibility, as it's still a branch meant for iteration!

**Review by AntonLydike — APPROVED** [review]
:+1:

**Pull request merged**